build.sh

```diff
@@ -508,6 +512,40 @@ build_hhvm() {
 }
 
 
+# autoconf-2.13 is needed to build spidermonkey
+build_autoconf() {
+    echo "\n===> Download and build autoconf-2.13\n"
+    if [ -d ${wrkdir}/autoconf-2.13 ]; then return; fi
+    cd ${wrkdir} || exit $?
+    wget http://ftp.gnu.org/gnu/autoconf/autoconf-2.13.tar.gz || exit $?
+    tar xfz autoconf-2.13.tar.gz || exit $?
+    cd autoconf-2.13
+    ./configure --prefix=${wrkdir}/autoconf-inst || exit $?
+    make install || exit $?
+    cd ${wrkdir}/autoconf-inst/bin
```

Comment:
I don't think it's worth being that fiddly, unless it turns out that we really need to do so. I mean, none of the other VMs is remotely fussy about this ATM.